src/ui/camera.js

```diff
@@ -33,11 +33,26 @@ import type {PaddingOptions} from '../geo/edge_insets';
  * @typedef {Object} CameraOptions
  * @property {LngLatLike} center The desired center.
  * @property {number} zoom The desired zoom level.
- * @property {number} bearing The desired bearing, in degrees. The bearing is the compass direction that
- * is "up"; for example, a bearing of 90° orients the map so that east is up.
- * @property {number} pitch The desired pitch, in degrees.
+ * @property {number} bearing The desired bearing in degrees. The bearing is the compass direction that
+ * is "up". For example, `bearing: 90` orients the map so that east is up.
+ * @property {number} pitch The desired pitch in degrees. The pitch is the angle towards the horizon
```

Comment:
"angle towards the horizon"

maybe it's just me, but that leaves me confused since an angle is measure between two things, so I always think of this as the angle between the camera and the map plane's normal vector. 

> `pitch: 0` results in a two-dimensional map,
￼> * as if your line of sight forms a perpendicular angle with the earth's surface.

what about adding "ie. looking straight down on the map", just a suggestion :shrug: